dev/Deployment/DeploymentManager.cpp

```diff
@@ -8,8 +8,12 @@
 #include <TerminalVelocityFeatures-DeploymentAPI.h>
 #include <Microsoft.Windows.ApplicationModel.WindowsAppRuntime.DeploymentManager.g.cpp>
 
+#include <WinBase.h>
 #include "WindowsAppRuntime-License.h"
 
+using namespace winrt;
+using namespace Windows::Foundation;
```

Comment:
added? This was a C++/WinRT implementation in 1.0, you just added the options object and initialize overload. These weren't declared before?